test/localhost.jl

```diff
@@ -0,0 +1,17 @@
+@testset "Localhost" begin
+    wmic_command = "wmic path win32_computersystemproduct get uuid"
+
+    windows_patch = @patch function Sys.iswindows()
+        return true
+    end
+
+    wmic_patch = @patch function read(command::Cmd, String)
+        return "UUID\nEC2D1284-E32E-FB5E-20E4-F43F6E01CA7A"
+    end
+
+    @testset "EC2 - Windows" begin
+        apply([windows_patch, wmic_patch]) do
+            @test localhost_is_ec2()
```

Comment:
Do we actually test this function on EC2 somewhere?

Reply:
I'd make an issue for this and we can leave this for now.

Reply:
https://github.com/JuliaCloud/AWSCore.jl/issues/90